### PR TITLE
cmd/lncli: Fix dropped error

### PR DESCRIPTION
**Pull request by alrs**
This PR fixes a dropped error in the `lncli` command.

### CODE REVIEW

File: cmd/lncli/commands.go [hunk 1]
```diff
@@ -2910,6 +2910,9 @@ func getChanInfo(ctx *cli.Context) error {
 		chanID = ctx.Int64("chan_id")
 	case ctx.Args().Present():
 		chanID, err = strconv.ParseInt(ctx.Args().First(), 10, 64)
+		if err != nil {
+			return err
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: Could you add some context to the error like so: `unable to parse chan id`

Comment (alrs) [reply cmd/lncli/commands.go]: Happily, made the change.

**Comment by alrs** [comment]
@wpaulino All good for merging?

**Comment by wpaulino** [comment]
Could you squash the commits into one please?

**Comment by alrs** [comment]
@wpaulino Squashed.

**Pull request merged**